### PR TITLE
Data acceptor server configured to work with AWS.

### PR DESCRIPTION
**Pull request by Shico-UA**
@TamoshaytisV @auraz 
The server accepts data from edx-platforms all over the world and displays the data on the world map.
Currently 2(two) edx-platform sending data on this server for testing purposes.
Link of the working server: http://acceptor-aws-env.hi8v7izgbx.eu-central-1.elasticbeanstalk.com/

### CODE REVIEW

File: .ebextensions/django.config [hunk 1]
```diff
@@ -0,0 +1,3 @@
+option_settings:
+  aws:elasticbeanstalk:container:python:
+    WSGIPath: acceptor/wsgi.py
```
Comment (auraz) [inline .ebextensions/django.config]: new line!

File: graph_creator/views.py [hunk 2]
```diff
@@ -73,10 +74,23 @@ def post(self, request, *args, **kwargs):
...
+                secret_token=secret_token,
+                platform_url=platform_url,
+                courses_amount=int(courses_amount),
+                students_amount=int(students_amount),
+                latitude=float(latitude),
+                longitude=float(longitude)
             )
+            if settings.DEBUG:
+                reverse_token = requests.post(
+                    # Local address of the edx-platform running within VM.
```
Comment (auraz) [inline graph_creator/views.py]: move url to settings, make setting external, etc

File: .ebextensions/django.config (hunk 1 above)
Comment (Shico-UA) [inline .ebextensions/django.config]: Done

File: graph_creator/views.py (hunk 2 above)
Comment (Shico-UA) [inline graph_creator/views.py]: Done

**Comment by auraz** [comment]
HTTP POST should answer correctly https://tools.ietf.org/html/rfc7231#section-4.3.3

File: graph_creator/views.py [hunk 3]
```diff
@@ -70,20 +70,38 @@ def post(self, request, *args, **kwargs):
...
                 students_amount=int(students_amount),
                 latitude=float(latitude),
                 longitude=float(longitude),
-                platform_url=platform_url
+                platform_url=platform_url,
+                site_name=site_name
             )
 
-        return redirect(reverse('graph_creator:index'))
+            return HttpResponse(status=200)
```
Comment (auraz) [inline graph_creator/views.py]: wrong indent?

File: graph_creator/views.py [hunk 4]
```diff
@@ -70,20 +70,38 @@ def post(self, request, *args, **kwargs):
...
+                requests.post(
+                    # Local IP address of the edx-platform running within VM.
+                    settings.EDX_PLATFORM_POST_URL_LOCAL, data={"secret_token": secret_token}
+                )
+                return HttpResponse(status=200)
+            else:
+                requests.post(
+                    str(platform_url) + '/acceptor_data/', data={"secret_token": secret_token}
+                )
+                return HttpResponse(status=200)
```
Comment (auraz) [inline graph_creator/views.py]: Why not created? (201)?

File: graph_creator/views.py (hunk 3 above)
Comment (auraz) [inline graph_creator/views.py]: do we have a number for updated?

File: graph_creator/views.py (hunk 4 above)
Comment (Shico-UA) [inline graph_creator/views.py]: Done

File: graph_creator/views.py (hunk 3 above)
Comment (Shico-UA) [inline graph_creator/views.py]: No, everything works ok.

Comment (Shico-UA) [inline graph_creator/views.py]: For update also used 200.

**Pull request merged**